simulation/__main__.py

```diff
@@ -97,10 +101,12 @@ def format_uuid(str):
     )
 
     parser.add_argument(
-        "--air-source-heat-pump-discount-factor-2022",
-        type=float,
-        default=0.1,
-        help="A factor by which current (2021) air source heat pump unit+install costs will have declined by, as of the end of 2022",
+        "--air-source-heat-pump-price-discount-date",
+        action="append",
+        type=map_string_to_datetime_float_tuple,
+        default=[],
```

Comment:
It would be a default of `2023-01-01:0.3` (the float is the amount by which the price is discounted, not the proportion of the original price). But the args get appended to this default, so that would be strange in the case we want to override it.